install.sh

```diff
@@ -1,15 +1,97 @@
 #!/usr/bin/env sh
 
 sudo apt-get update
-sudo apt-get install -y ccache cmake clang-format python3-pip python3-setuptools libgmp3-dev parallel
-
+sudo apt-get install -y ccache cmake clang clang-format-7 python3-pip python3-setuptools libgmp3-dev g++ parallel
 python3 -m pip install --upgrade wheel
-python3 -m pip install --upgrade conan
 python3 -m pip install --upgrade trio
 
-conan profile new default --detect
-conan profile update settings.compiler.libcxx=libstdc++11 default
+if [ "$1" = "USE_CONAN" ]; then
```

Comment:
This doesn't look correct. Shouldn't it be `==` instead of `=` ? 
Also, shouldn't the value be either `ON` or `OFF` based on configuration. I think it's better to use `-n` and pass an unset variable when we don't wan to use conan, as this allows all the booleanesque functionality of CMAKE in it's variables (ON, OFF, TRUE, FALSE, 0, 1, etc...)

How did you test this?

Reply:
Fixed